### PR TITLE
Use original ordering and grouping for the 25 exemplars

### LINKED ISSUE #844: Original 25 exemplars appear in alphabetical order, not original priority order
Be nice if the numbering [on this list](https://govuk-digital-services.herokuapp.com/original-25-exemplars) matched that on [the original list](https://www.gov.uk/government/publications/gds-transformation-programme-2013-to-2015/digital-transformation-exemplar-services).

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
Fixes #844.

- Maintain list grouping and numbering used by the original Digital Transformation programme
- Add logo used for the programme
- Update layout to use two columns:

  Screenshot of updated page.

I also did a bit of detective work to find the final 4 missing services:

- ​5. _Search for property information from HM Land Registry_
- ​15. _Check or update your company car tax_
- ​16. _Pay your Self Assessment tax bill_
- ​18. _Agent services account_

### CODE REVIEW

**Comment by frankieroberto** [comment]
Oh, this is nice! I was never sure whether anyone else would be interested in this list, but it felt like a nice bit of history to me!

File: start.js [hunk 1]
```diff
@@ -248,7 +252,35 @@ app.get('/top-75', function(req, res) {
 });
 
 app.get('/original-25-exemplars', function(req, res) {
-  res.render('original-25-exemplars.html')
+  res.render('original-25-exemplars.html', {
+    exemplars: [
+      "Register to vote",
```
[the remark this answers is not in the document]
Comment (paulrobertlloyd) [reply start.js:257]: That was the original plan, but looks like `slug` is not in the source data? Or am I missing something?

File: start.js [hunk 2]
```diff
@@ -228,6 +228,10 @@ env.addFilter('formatdate', function(str) {
   return `${date.getDate()} ${monthNames[date.getMonth()]} ${date.getFullYear()}`
 });
 
+env.addFilter('where', function (array, key, value) {
```
Comment (frankieroberto) [inline start.js:231]: Could call this filter “find” to match the JavaScript function? “Where” to me implies “get all matching” but the find function only returns the first one.

**Comment by frankieroberto** [comment]
PS I didn’t think the order of the original 25 was actually significant / prioritised? But agree it’s still nice to match the order.

File: start.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (paulrobertlloyd) [reply start.js:257]: Oh wait, no, I’m talking rubbish. My original implementation didn’t have slug, but the revision approach does, and I’m actually using it! :facepalm:

**Comment by paulrobertlloyd** [comment]
Updated to group services under the original organisation they were developed under. We don’t have the ‘Department for Business Innovation & Skills’ (since renamed and reorganised a good few times), and the services listed refer the relevant executive agencies instead, so have hard-coded the organisation name for that grouping of exemplar services.

**Review by frankieroberto — APPROVED** [review]
Nice, thanks!

**Pull request merged**